src/popup/popup.less

```diff
@@ -50,3 +50,17 @@ body {
 .info {
     white-space: nowrap;
 }
+
+@media (prefers-color-scheme: dark) {
+    body {
+        background: black;
```

Comment:
Could the dark theme colours please match the dark theme colours for browserpass.